Pseudo_experiments/nbconverted/generate_E_GEOD_51409_template_experiment.py

```diff
@@ -130,6 +133,8 @@
 
 
 # ## Process data
+# 
+# Notice: Originally the expression data was 0-1 normalized for use in training the VAE, however when we performed differential expression analyses we found that the normalized data had reduced variance that resulted in an inconsistency between the number of DEGs found compared to the publication. Therefore, we are re-scaling our normalized data to be in the original range of data.
```

Comment:
Just a curiosity thing: is this repo going to be public when the manuscript is published? I'm wondering if your code will have to be understandable to random passersby like me or if the documentation is just for future you and other Greene lab members who might want to continue the project. Either way, I think you're in good shape.

Reply:
This repo is already public, but will be highlighted in the manuscript that we're working on. I anticipate outside users may want to use the simulation notebooks, but not these type of followup experiments that more to support using the VAE to simulate data.  So I'm not as concerned about these scripts in that sense.